presto-proxy/src/test/resources/shared-secret-file

```diff
@@ -0,0 +1 @@
+# for test purposes only
```

Comment:
we should not need a dummy file for this. a /dev/null, a pom.xml?

@electrum any preference?

Reply:
Let's use `/dev/null` everywhere for this.